settings/global.mdx

```diff
@@ -462,7 +462,11 @@ Example:
     Enables a rating system for users to indicate whether the page has been helpful
     </ResponseField>
     <ResponseField name="suggestEdit" type="boolean" default={false}>
-    Enables a button to allow users to suggest edits via pull requests
+    Enables a button to allow users to suggest edits via pull requests for public repositories.
+
+    <Warning>
+      If your docs repo is private, `suggestEdit` will not work.
+    </Warning>
```

Comment:
unfortunately this looks jank

![CleanShot 2024-12-05 at 10 39 59@2x](https://github.com/user-attachments/assets/e066d03a-728f-49b5-bb46-83e40e78dbfa)

Reply:
fixed thru merging a styling pr on `mint`